src/viur/core/modules/translation.py

```diff
@@ -126,21 +126,23 @@ class Translation(List):
     kindName = KINDNAME
 
     def adminInfo(self):
-        admin_info = {
-            "name": translate("translations"),
-            "icon": "translate",
-            "views": [
-                {
-                    "name": translate("core.translations.view.missing",
-                                      "Missing translations for {{lang}}",
-                                      )(lang=lang),
-                    "filter": {
-                        "translations_missing": lang,
-                    },
-                }
-                for lang in conf.i18n.available_dialects
-            ],
-        }
+        if admin_info := conf.translation_admin_info:
```

Comment:
If the config is empty (bc I just want to use the config as it is defined in this method) that branch would be skipped.

I think this merging is in general kinda hackish. 

monkeypatching `Translation.adminInfo = { ... }` would btw work as well.